charts/openshift-metering/templates/presto/presto-coordinator-config.yaml

```diff
@@ -13,13 +13,29 @@ data:
     node.environment=production
 
   config.properties: |
-    http-server.http.port=8080
-    jmx.rmiserver.port=8081
-    jmx.rmiregistry.port=8081
     coordinator=true
+    node-scheduler.include-coordinator={{ .Values.presto.spec.presto.config.nodeSchedulerIncludeCoordinator }}
```

Comment:
Honestly this could probably be hardcoded and removed from `values.yaml`

Reply:
Shoot my bad. I meant to put this on `discovery.uri`, sorry.

Reply:
Updated in the last force-push